lib/ebpd_utils.h

```diff
@@ -0,0 +1,30 @@
+#ifndef EBPD_UTILS_H_
+#define EBPD_UTILS_H_
+
+#ifdef __cplusplus
+extern "C"
```

Comment:
Why is ebpd_utils.h written in C? Do we expect to call it from pure C code? Even if this is the case you can declare extern "C" functions that call an underlying C++ implementation.

Reply:
This file is meant to be a wrapper around the libbpf APIs. Hence kept it in C. For now I think we can keep it in C unless there is a pressing need.